src/OperationResponse.ts

```diff
@@ -0,0 +1,9 @@
+import Logger from "./logger"
```

Comment:
This is totally a judgement call, but I am less of a fan of pulling out helpers for situations like this as every instance of a use case often want to be or grow in different directions.  What do you think about inlining this and keeping the  typing scoped to the feature?

Reply:
I agree with john here, these tend to drive toward the too generic side when they are used across the board and maybe they do so in detriment of the actual usefulness they can get.